.github/workflows/unit_tests.yaml

```
@@ -39,7 +39,7 @@ jobs:
         python-version: 3.9
     - name: Install Python packages
       run: |
-        pip install --upgrade pip six setuptools flake8 flake8-import-order mypy>=0.800 black types-six types-python-dateutil
+        pip install --upgrade pip six setuptools flake8 isort mypy>=0.800 black types-six types-python-dateutil
```

```suggestion
        pip install --upgrade pip six setuptools flake8 isort>=4.3.5 mypy>=0.800 black types-six types-python-dateutil
```
This is the first version of isort that supports pyproject.toml. Not sure if there are any newer features we're using, but an incorrect minimum version is better than no minimum version.

---

done